--- config.h.in	2020-05-27 18:13:34.342484711 +1000
+++ config.h_new.in	2020-05-27 18:12:44.323419000 +1000
@@ -155,7 +155,7 @@
 #undef HAVE_GDLIB
 
 /* Define if the GD library supports FONTCONFIG */
-#undef HAVE_GD_FONTCONFIG
+#define HAVE_GD_FONTCONFIG
 
 /* Define if the GD library supports FREETYPE */
 #undef HAVE_GD_FREETYPE
@@ -173,7 +173,7 @@
 #undef HAVE_GD_OPENPOLYGON
 
 /* Define if the GD library supports PNG */
-#undef HAVE_GD_PNG
+#define HAVE_GD_PNG
 
 /* Define if the GD library supports XPM */
 #undef HAVE_GD_XPM
@@ -233,7 +233,7 @@
 #undef HAVE_LIBDLLOADER
 
 /* Define if you have the GD library */
-#undef HAVE_LIBGD
+#define HAVE_LIBGD
 
 /* Define to 1 if you have the `ltdl' library (-lltdl). */
 #undef HAVE_LIBLTDL
