From ea13be764580ce5d7505c5753e41e2f063808525 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 15 Feb 2021 18:07:41 -0500
Subject: [PATCH 2/3] Remove requirement of hard coded DLLs in PATH

---
 cmake/FindCAIRO.cmake      | 5 -----
 cmake/FindPANGOCAIRO.cmake | 8 --------
 2 files changed, 13 deletions(-)

diff --git a/cmake/FindCAIRO.cmake b/cmake/FindCAIRO.cmake
index 5d706b614..bf944a64e 100644
--- a/cmake/FindCAIRO.cmake
+++ b/cmake/FindCAIRO.cmake
@@ -15,11 +15,6 @@ if(WIN32)
     CAIRO_INCLUDE_DIR
 
     CAIRO_LIBRARY
-
-    CAIRO_RUNTIME_LIBRARY
-    EXPAT_RUNTIME_LIBRARY
-    FONTCONFIG_RUNTIME_LIBRARY
-    PIXMAN_RUNTIME_LIBRARY
   )
 
   set(CAIRO_INCLUDE_DIRS ${CAIRO_INCLUDE_DIR})
diff --git a/cmake/FindPANGOCAIRO.cmake b/cmake/FindPANGOCAIRO.cmake
index e045e8ff6..7f3574a87 100644
--- a/cmake/FindPANGOCAIRO.cmake
+++ b/cmake/FindPANGOCAIRO.cmake
@@ -35,14 +35,6 @@ if(WIN32)
     PANGO_LIBRARY
     PANGOCAIRO_LIBRARY
     HARFBUZZ_LIBRARY
-
-    GLIB_RUNTIME_LIBRARY
-    GOBJECT_RUNTIME_LIBRARY
-    HARFBUZZ_RUNTIME_LIBRARY
-    PANGO_RUNTIME_LIBRARY
-    PANGOCAIRO_RUNTIME_LIBRARY
-    PANGOFT_RUNTIME_LIBRARY
-    PANGOWIN_RUNTIME_LIBRARY
   )
 
   set(PANGOCAIRO_INCLUDE_DIRS
-- 
2.31.1

