From ebb37f3377f6f3844a0c9f397b00e4e8600b7f64 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 15 Feb 2021 18:07:41 -0500
Subject: [PATCH 2/3] Remove requirement of hard coded DLLs in PATH

---
 cmake/FindCAIRO.cmake      | 5 -----
 cmake/FindPANGOCAIRO.cmake | 8 --------
 2 files changed, 13 deletions(-)

diff --git a/cmake/FindCAIRO.cmake b/cmake/FindCAIRO.cmake
index a92916eb9..77e4895eb 100644
--- a/cmake/FindCAIRO.cmake
+++ b/cmake/FindCAIRO.cmake
@@ -15,11 +15,6 @@ if(WIN32)
     CAIRO_INCLUDE_DIR
 
     CAIRO_LIBRARY
-
-    CAIRO_RUNTIME_LIBRARY
-    EXPAT_RUNTIME_LIBRARY
-    FONTCONFIG_RUNTIME_LIBRARY
-    PIXMAN_RUNTIME_LIBRARY
   )
 
   set(CAIRO_INCLUDE_DIRS ${CAIRO_INCLUDE_DIR})
diff --git a/cmake/FindPANGOCAIRO.cmake b/cmake/FindPANGOCAIRO.cmake
index b92e5bea9..76e8ede19 100644
--- a/cmake/FindPANGOCAIRO.cmake
+++ b/cmake/FindPANGOCAIRO.cmake
@@ -35,14 +35,6 @@ if(WIN32)
     PANGO_LIBRARY
     PANGOCAIRO_LIBRARY
     HARFBUZZ_LIBRARY
-
-    GLIB_RUNTIME_LIBRARY
-    GOBJECT_RUNTIME_LIBRARY
-    HARFBUZZ_RUNTIME_LIBRARY
-    PANGO_RUNTIME_LIBRARY
-    PANGOCAIRO_RUNTIME_LIBRARY
-    PANGOFT_RUNTIME_LIBRARY
-    PANGOWIN_RUNTIME_LIBRARY
   )
 
   set(PANGOCAIRO_INCLUDE_DIRS
-- 
2.33.1

