From 28c77ce15c55ed866065769b42e9d37d3bd65195 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 2 Feb 2025 18:04:50 +0000
Subject: [PATCH 5/5] Don't rely on pkgconfig for GTS on Windows

---
 cmake/FindGTS.cmake | 31 ++++++++++++++++++++++++++-----
 1 file changed, 26 insertions(+), 5 deletions(-)

diff --git a/cmake/FindGTS.cmake b/cmake/FindGTS.cmake
index cc24e45a6..aa0248055 100644
--- a/cmake/FindGTS.cmake
+++ b/cmake/FindGTS.cmake
@@ -19,19 +19,40 @@ if(MINGW)
     ${GLIB_RUNTIME_LIBRARIES}
   )
 elseif(WIN32)
-  find_package(GLIB)
+  find_path(GTS_INCLUDE_DIR gts.h)
+  find_path(GLIB_INCLUDE_DIR glib.h PATH_SUFFIXES glib-2.0)
+  find_path(GLIBCONFIG_INCLUDE_DIR glibconfig.h
+            PATH_SUFFIXES glib-2.0/include lib/glib-2.0/include)
+
+  find_library(GTS_LIBRARY NAMES gts)
+  find_library(GLIB_LIBRARY NAMES glib-2.0)
 
-  find_program(GTS_RUNTIME_LIBRARY NAMES gts.dll)
+  find_program(GTS_RUNTIME_LIBRARY NAMES gts.dll libgts-0-7-5.dll)
+  find_program(GLIB_RUNTIME_LIBRARY NAMES glib-2.dll libglib-2.0-0.dll)
 
   find_package_handle_standard_args(GTS DEFAULT_MSG
-    GTS_INCLUDE_DIRS
+    GTS_INCLUDE_DIR
+    GLIB_INCLUDE_DIR
+    GLIBCONFIG_INCLUDE_DIR
 
-    GTS_LINK_LIBRARIES
+    GTS_LIBRARY
+    GLIB_LIBRARY
+  )
+
+  set(GTS_INCLUDE_DIRS
+    ${GTS_INCLUDE_DIR}
+    ${GLIB_INCLUDE_DIR}
+    ${GLIBCONFIG_INCLUDE_DIR}
+  )
+
+  set(GTS_LINK_LIBRARIES
+    ${GTS_LIBRARY}
+    ${GLIB_LIBRARY}
   )
 
   set(GTS_RUNTIME_LIBRARIES
     ${GTS_RUNTIME_LIBRARY}
-    ${GLIB_RUNTIME_LIBRARIES}
+    ${GLIB_RUNTIME_LIBRARY}
   )
 else()
   find_package_handle_standard_args(GTS DEFAULT_MSG
-- 
2.47.1

