From 346ff861e8aeaf3de5c511fcdf255f00a4267d28 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 15 Feb 2021 15:20:40 -0500
Subject: [PATCH 5/7] Use version agnostic python binary

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 171e38e54..5879249e6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -112,7 +112,7 @@ if (DEFINED VERSION)
     set(GRAPHVIZ_VERSION_STRING "${VERSION}")
 else()
     execute_process(
-        COMMAND           python3 gen_version.py
+        COMMAND           python gen_version.py
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
         RESULT_VARIABLE   gen_version_result
         OUTPUT_VARIABLE   GRAPHVIZ_VERSION_STRING
@@ -130,7 +130,7 @@ if (DEFINED DATE)
 else()
     # Set GRAPHVIZ_VERSION_BUILD to time of last commit, or to 0 if that fails.
     execute_process(
-        COMMAND           python3 gen_version.py --committer-date-graphviz
+        COMMAND           python gen_version.py --committer-date-graphviz
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
         RESULT_VARIABLE   gen_date_result
         OUTPUT_VARIABLE   GRAPHVIZ_VERSION_BUILD
-- 
2.30.1

